rr_platform/msg/chassis_state.msg

```diff
@@ -0,0 +1,5 @@
+# Message for defining the state of the car
+Header header           # header for when the message is published
+bool estop              # whether the estop is on or not
```

Comment:
With boolean flags like this, it is very helpful to specify in the comments the specific meaning of each state (true/false).
Generic descriptions like this can be confusing for other developers. For example, here, does the estop being "on" mean the car is enabled or disabled? It is easy to argue for either interpretation given just this description.